### DISCUSSION TITLE
Testflight invite

### DISCUSSION THREAD

**Discussion by deepu** [olmps/memo#102] [General (English)]
Hey,

Awesome project. 
IF you need someone to test and report any issues , I will be glad to do us.
A TestFlight invite possible ?

**Comment by matuella** [comment]
Thanks a lot @deepu!

We will be adding a TestFlight and Google Play links soonish, I'll make sure to update this discussion once we have it!

Also, there is one relevant info: this first alpha version will be supporting only ptBR language, but we are looking forward to release an english version!
